bchain/coins/eth/ethrpc.go

```diff
@@ -1219,20 +1219,26 @@ func (b *EthereumRPC) EthereumTypeGetNonce(addrDesc bchain.AddressDescriptor) (u
 			addrDesc,
 			"pending",
 		)
-		if err != nil {
+		if err == nil && result != "" {
+			useAlternative = true
+			glog.Errorf("Alternative provider success for eth_getTransactionCount")
```

Comment:
I would not log success, it will fill the log with unnecessary lines. Maybe keep for testing in the branch but would not merge it...